ios/Exponent/Kernel/AppLoader/EXAppLoaderExpoUpdates.m

```diff
@@ -537,6 +538,19 @@ - (nullable EXManifestsManifest *)_processManifest:(EXManifestsManifest *)manife
     if (![mutableManifest[@"isVerified"] boolValue] && (EXEnvironment.sharedEnvironment.isManifestVerificationBypassed || [EXAppLoaderExpoUpdates _isAnonymousExperience:manifest])) {
       mutableManifest[@"isVerified"] = @(YES);
     }
+    
+    // when the manifest is not verified at this point, make the scope key and id a random value
+    if (![mutableManifest[@"isVerified"] boolValue]) {
+      NSString *randomValue = [[NSUUID UUID] UUIDString];
+      if ([manifest isKindOfClass:EXManifestsNewManifest.class]) {
+        NSMutableDictionary *mutableExtra = [mutableManifest[@"extra"] mutableCopy];
+        mutableExtra[@"scopeKey"] = randomValue;
+        mutableManifest[@"extra"] = mutableExtra;
+      } else {
+        mutableManifest[@"scopeKey"] = randomValue;
```

Comment:
Might be useful to make unverified scope keys easily identifiable (ex. if they show up during debugging why a project's scoped data is no longer accessible) by prefixing them with `usk-` or something (we were thinking to use `sk-` for verified scope keys for new projects).